docs/preview/03-Features/powershell/azure-api-management.md

```diff
@@ -192,6 +192,201 @@ PS> Create-AzApiManagementUserAccount `
 # Account has been created for $FirstName $LastName ($mailAddress) for Azure API Management service '$ServiceName' in resource group '$ResourceGroupName'
 ```
 
+## Applying user configuration to an Azure API Management service from a configuration file
+
+Apply user configuration to an existing Azure API Management instance. You can create or update users and assign groups and/or subscriptions to these users.
+
+| Parameter                         | Mandatory | Description                                                                                                                                                         |
+| --------------------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `ResourceGroupName`               | yes       | The resource group containing the Azure API Management instance                                                                                                     |
+| `ServiceName`                     | yes       | The name of the Azure API Management instance located in Azure                                                                                                      |
+| `ConfigurationFile`               | yes       | Path to the JSON Configuration file containing the user configuration                                                                                               |
+| `StrictlyFollowConfigurationFile` | no        | Indicates whether the configuration file should strictly be followed, for example remove the user from groups not defined in the configuration file                 |
```

Comment:
Maybe this can be a switch instead of a boolean? Thinking about that this functionality is a bit `Upsert` and `Append`, right? Just thinking out-load here. It is already a very descriptive switch that would probably be clear in every situation. Good job on that!

Reply:
It actually already is a switch, just not properly documented, should be better now, see [`7780677` (#422)](https://github.com/arcus-azure/arcus.scripting/pull/422/commits/7780677944b2ce2c27779a30e89c07c1ae9ade90)